app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -331,4 +354,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    @Override
+    public void onSpokenText(String searchInput) {
```

Comment:
nit: Can we move this under the voiceEntry() block to keep code working towards the same goal together? Better for organization.